app/src/main/java/app/gamenative/db/converters/GOGConverter.kt

```
@@ -0,0 +1,21 @@
+class GOGConverter {
+
+    @TypeConverter
+    fun fromStringList(value: List<String>): String {
+        return Json.encodeToString(value)
+    }
+
+    @TypeConverter
+    fun toStringList(value: String): List<String> {
+        return try {
+            Json.decodeFromString<List<String>>(value)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)